studentClass/src/studentclass/StudentClass.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package studentclass;
+
+/**
+ *
+ * @author DEE
+ */
+public class StudentClass {
+
+    /**
+     * @param args the command line arguments
+     */
+    static class Student {

```

Comment:
:-1: There should only be one `Student` class. I'm not sure why you have multiple ones.